### PR TITLE
Added Case Studies page in Overview Section 

### PR DESCRIPTION
**Pull request by TamannaVerma99**
**What kind of change does this PR introduce?**
Added a new page in Overview section to show the case studies published in the blog. Currently, since I don't have access to the data file that has information related to blogs published, I have created a sample file of my own named `casestudies.json` with the same data format as mentioned in the issue and linked it to the card component. This makes the JSON file easily replaceable with the type of data we'll be using.

**Issue Number:**

-  Related to #440  

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by benjagm** [comment]
@TamannaVerma99 Hi reviewed the PR and decided to push some changes to accelerate the process. The only pending thing on your side is to adjust the generic card component to make sure the logos shows properly. I like the logos much bigger with the right proportion bacause some of the logos show deformed.

What I fixed:

- Modified the json data to remove posts unrelated to case studies and updated the logos.
- Added all the logos.
- Modified how you was using the card component to use the image property to show the logo, instead of the icon as you was using. Now the behaviour is correct.

You did an amazing job and we are very very close.
Congrats!!

**Review by benjagm — CHANGES_REQUESTED** [review]
Nice job. We are very very close.

See my review comments [here](https://github.com/json-schema-org/website/pull/540#issuecomment-2001921707)

**Comment by TamannaVerma99** [comment]
> Nice job. We are very very close.
> 
> See my review comments [here](https://github.com/json-schema-org/website/pull/540#issuecomment-2001921707)

Thanks :) I will do the required changes asap.

File: components/Cardforcasestudies.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <div className='flex justify-center '>
+        {image && <img src={image} className='h-32 w-36 p-2' />}
+      </div>
+      <div className='flex flex-row items-start mb-6'>
+        {icon && (
+          <span className='mr-6 flex h-14 w-14 flex-shrink-0 items-center justify-center rounded-lg border bg-blue-200 px-3 text-gray-900'>
+            <img src={icon} alt={title} className='h-full w-full' />
+          </span>
+        )}
+        <p className='mt-1 items-center text-[0.9rem] font-bold text-gray-900'>
```
Comment (Michael-Obele) [inline components/Cardforcasestudies.tsx:24]: Hey there! Great work on the code! I noticed the header seemed a bit small. Might we consider text-h5 or text-h6 for better readability?
```suggestion
        <p className='mt-1 items-center text-h5 font-bold text-gray-900'>
```

Comment (TamannaVerma99) [reply components/Cardforcasestudies.tsx:24]: Hey! Thanks for the feedback .I will work on it. Thanks:)

**Comment by Michael-Obele** [comment]
Hi everyone,
I realize now that the new card component was likely created to address the issue of long titles exceeding the character limit. This is something I hadn't considered initially.

I'd be happy to adjust the existing card component by adding a conditional statement. This would check the word count and automatically adjust the text size to be larger for shorter content and smaller for longer content.

To keep things moving smoothly, I can either open a separate issue for this or jump right in and make the fix here in the PR.  Just let me know which approach you prefer!

**Comment by VivekJaiswal18** [comment]
> Hi everyone, I realize now that the new card component was likely created to address the issue of long titles exceeding the character limit. This is something I hadn't considered initially.
> 
> I'd be happy to adjust the existing card component by adding a conditional statement. This would check the word count and automatically adjust the text size to be larger for shorter content and smaller for longer content.
> 
> To keep things moving smoothly, I can either open a separate issue for this or jump right in and make the fix here in the PR. Just let me know which approach you prefer!

But if we keep the title text size variable will it not make the UI inconsistent, if in the same page we have heading with variable length

**Comment by Michael-Obele** [comment]
> > Hi everyone, I realize now that the new card component was likely created to address the issue of long titles exceeding the character limit. This is something I hadn't considered initially.
> > I'd be happy to adjust the existing card component by adding a conditional statement. This would check the word count and automatically adjust the text size to be larger for shorter content and smaller for longer content.
> > To keep things moving smoothly, I can either open a separate issue for this or jump right in and make the fix here in the PR. Just let me know which approach you prefer!
> 
> But if we keep the title text size variable will it not make the UI inconsistent, if in the same page we have heading with variable length

You raise a good point about UI consistency.  While increasing the base font size slightly could help with longer titles, perhaps a maximum character limit or using ellipsis for overflow could be additional solutions to explore.

**Comment by Michael-Obele** [comment]
Although I don't see any issue with having a separate card component as we have now, this is only a suggestion.

**Comment by TamannaVerma99** [comment]
> Although I don't see any issue with having a separate card component as we have now, this is only a suggestion.

Hey! Actually I have also been working on welcome page ,for there too I have same issues if my title is even "Getting Started" still its covering two lines so it would be really great if we find any possible solution for it .I think we can also pass title heading by props.

**Comment by benjagm** [comment]
This code has been already merged in the dev branch. Awesome work!

File: components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  title: string;
+  body: string;
+  icon?: string;
+  link?: string;
+  image?: string;
+}
+
+const CardBody = ({ title, body, icon, link, image }: CardProps) => {
+  return (
+    <div className='group relative h-full w-full max-w-lg rounded-lg border border-gray-200 bg-white p-6 px-12 shadow-3xl transition-colors delay-[150ms] ease-in-out hover:bg-slate-100'>
```
Comment (VivekJaiswal18) [inline components/Card.tsx:14]: @TamannaVerma99 Please add `dark:text-black` in the div. As in dark mode the text is not visible.
image

**Pull request closed without merging**